components/schema-migrator/migrations/director/202002071200_add_package_definitions.up.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE packages (
+    id UUID PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
```

Comment:
Oh, one thing I forgot - not nullable `applicationID` for the packages (as a foreign key)